packages/@lwc/integration-karma/test/rendering/version-mismatch/index.spec.js

```diff
@@ -48,9 +48,27 @@ describe('compiler version mismatch', () => {
         });
 
         afterEach(() => {
+            process.env.SKIP_LWC_VERSION_MISMATCH_CHECK = 'false';
             detachReportingControlDispatcher();
         });
 
+        it('skip warning during local dev', () => {
+            process.env.SKIP_LWC_VERSION_MISMATCH_CHECK = 'true';
+            function tmpl() {
+                return [];
+                /*LWC compiler v123.456.789*/
+            }
+
+            expect(() => {
+                registerTemplate(tmpl);
+            }).not.toLogErrorDev(
+                new RegExp(
+                    `LWC WARNING: current engine is v${process.env.LWC_VERSION}, but template was compiled with v123.456.789`
+                )
+            );
```

Comment:
```suggestion
            }).not.toLogErrorDev(/v123\.456\.789/);
```
Checking that we're not logging such a long message feels easy to get an accidental false positive, if the error text ever changes. We should just check for keywords to be more flexible.

Reply:
sure I can make it more generic